src/download.rs

```diff
@@ -35,13 +36,19 @@ pub struct DownloadCfg {
     pub name: String,
     pub target: TargetTriple,
     pub version: Version,
+    pub date: Option<Date>,
```

Comment:
Maybe we should use a `PackageVersion` rather than separate `version` and `date` fields here?